Formula/spades.rb

```diff
@@ -0,0 +1,27 @@
+class Spades < Formula
+  desc "SPAdes: de novo genome assembly"
```

Comment:
this is circumventing the audit about descriptions not starting with formula name

Reply:
Fixed.